zeebe/broker/src/main/java/io/camunda/zeebe/broker/system/configuration/ClusterCfg.java

```diff
@@ -40,6 +45,15 @@ public final class ClusterCfg implements ConfigurationEntry {
           + " quorum = floor(replication factor / 2) + 1. In this current case the quorum will be"
           + " quorum = {}. If you want to ensure high fault-tolerance and availability,"
           + " make sure to use an odd replication factor.";
+
+  private static final String INITIAL_CONTACT_POINTS_ERROR_MSG =
+      "Initial contact points must be configured when cluster size is greater than 1. "
+          + "Please configure '"
+          + LEGACY_INITIAL_CONTACT_POINTS_PROPERTY
+          + "' or '"
+          + UNIFIED_INITIAL_CONTACT_POINTS_PROPERTY
+          + "'.";
```

Comment:
🙃 Is there any use in providing the legacy property here? Even if there is, it should probably be the second option and clearly marked with `(legacy)` or something.